server/handler/version.go

```diff
@@ -3,13 +3,31 @@ package handler
 import (
 	"net/http"
 
+	bblfsh "gopkg.in/bblfsh/client-go.v2"
+
 	"github.com/src-d/gitbase-playground/server/serializer"
+	"github.com/src-d/gitbase-playground/server/service"
 )
 
 // Version returns a function that returns a *serializer.Response
-// with a current version of server
-func Version(version string) RequestProcessFunc {
+// with a current version of server and dependencies
+func Version(version, bbblfshServerURL string, db service.SQLDB) RequestProcessFunc {
 	return func(r *http.Request) (*serializer.Response, error) {
-		return serializer.NewVersionResponse(version), nil
+		cli, err := bblfsh.NewClient(bbblfshServerURL)
+		if err != nil {
+			return nil, err
+		}
+
+		resp, err := cli.NewVersionRequest().Do()
+		if err != nil {
+			return nil, err
```

Comment:
So this will return an error if bblfsh is not reachable? What do you think about doing the same as gitbase, returning "unknown"? This way in the interface you could quickly see if one of the services is down.

Reply:
I thought about it but wasn't sure it's correct behavior. But if you think the same, I'm changing.